definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-journey-prod.json

```diff
@@ -194,5 +194,12 @@
     "CaseEventID": "solicitorDraftAOS",
     "UserRole": "[RESPSOLICITOR]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "18/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorUpdateAOS",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "R"
```

Comment:
It changes prod xlsx without releasing this feature...

Reply:
Exactly. If there is a resp-solicitor in prod (I doubt it), they will no longer be able to trigger this until we go live with this feature.